lab-taylor/app/component/app-gallery/app-gallery.html

```diff
@@ -0,0 +1,13 @@
+<section class="app-gallery">
+  <ul>
+    <app-gallery-item-text ng-repeat="item in appGalleryCtrl.images" title="{{item.title}}" desc="{{item.desc}}" img-src="{{item.imgSrc}}">
```

Comment:
I like your approach here, makes a lot of sense to me. Just a suggestion, since you're passing along a bunch of properties from the same object here, you'd type less if you just pass the object: `image="{{item}}"`